### PR TITLE
Flex usage and specs pages added to the site

### LINKED ISSUE #1472: Flex: Usage & Specifications

### PR DESCRIPTION
**Pull request by raquelarrojo**
**Checklist**
_(Check off all the items before submitting)_
- [x] Build process is done without errors and all tests pass in `/lib` directory.
- [x] Self-reviewed the code prior to submitting.
- [x] Meets accessibility standards.
- [x] Added/updated documentation to `/website` as needed.
- [x] Added/updated tests as needed.

**Purpose**
Add Flex usage and specs page to the site.

**Description**
Added Flex usage and specs page to the site.

**Closes #1472**

### CODE REVIEW

File: website/screens/components/flex/specs/FlexSpecsPage.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+const FlexSpecsPage = () => {
+  return (
+    <DxcFlex direction="column" gap="4rem">
+      <QuickNavContainerLayout>
+        <QuickNavContainer
+          sections={sections}
+          startHeadingLevel={2}
+        ></QuickNavContainer>
+      </QuickNavContainerLayout>
+      <DocFooter githubLink="https://github.com/dxc-technology/halstack-react/blob/master/website/screens/components/file-input/specs/FileInputSpecsPage.tsx" />
```
Comment (Lisa159) [inline website/screens/components/flex/specs/FlexSpecsPage.tsx:41]: Change the link from `file-input` to `flex`.

File: website/screens/components/flex/usage/FlexUsagePage.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    title: "Flexbox Container",
+    content: (
+      <>
+        <DxcParagraph>
+          An area of a document laid out using flexbox is called a flex
+          container.
+        </DxcParagraph>
+        <DxcBulletedList>
+          <DxcBulletedList.Item>
+            Items display in a row (the flex-direction property&#39;s default is
```
Comment (Lisa159) [inline website/screens/components/flex/usage/FlexUsagePage.tsx:72]: Maybe to put `The items` like in the following points?

File: website/screens/components/flex/usage/FlexUsagePage.tsx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+          <DxcBulletedList.Item>
+            The main axis is defined by flex-direction, which has four possible
+            values: <Code>row</Code>, <Code>row-reverse</Code>,{" "}
+            <Code>column</Code> and <Code>column-reverse</Code>.
+          </DxcBulletedList.Item>
+          <DxcBulletedList.Item>
+            The cross axis runs perpendicular to the main axis.
+            <DxcBulletedList type="circle">
+              <DxcBulletedList.Item>
+                If the flex-direction (main axis) is set to row or row-reverse
```
Comment (Lisa159) [inline website/screens/components/flex/usage/FlexUsagePage.tsx:31]: In the line 24 it is used the component `Code` for the values. I would continue use it in the rest when referring to this values as in this line 31.

File: website/screens/components/flex/usage/FlexUsagePage.tsx [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+                cross axis runs down the rows.
+              </DxcBulletedList.Item>
+            </DxcBulletedList>
+          </DxcBulletedList.Item>
+        </DxcBulletedList>
+      </>
+    ),
+  },
+  {
+    title: "Start and End",
```
Comment (Lisa159) [inline website/screens/components/flex/usage/FlexUsagePage.tsx:45]: In other components we are not using upper case except in the first word, like we do with `Design tokens` section. Maybe to change the titles to lower case?

File: website/screens/components/flex/usage/FlexUsagePage.tsx (hunk 3 above)
Comment (raquelarrojo) [reply website/screens/components/flex/usage/FlexUsagePage.tsx:31]: I agree in the others, but not here. In the Axes section we are defining the possible values of `direction` prop, so it makes sense to remark them with the `Code`. But here we are only talking about it, I think it makes no sense to remark it. Like in the Flexbox container section, that we are not remarking it.

File: website/screens/components/flex/usage/FlexUsagePage.tsx [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+    content: (
+      <>
+        <DxcParagraph>
+          The flexbox makes no assumption about the writing mode of the
+          document. Left to right or right to left can be used depending, for
+          example, on the language used. A start and end edge is used to refer
+          to the direction of the placement of elements.
+        </DxcParagraph>
+        <DxcParagraph>
+          For example, If the flex-direction is row and the language is English,
```
[the remark this answers is not in the document]
Comment (raquelarrojo) [reply website/screens/components/flex/usage/FlexUsagePage.tsx:55]: I agree in the others, but not here. In the Axes section we are defining the possible values of `direction` prop, so it makes sense to remark them with the `Code`. But here we are only talking about it, I think it makes no sense to remark it. Like in the Flexbox container section, that we are not remarking it.

File: website/screens/components/flex/specs/FlexSpecsPage.tsx (hunk 1 above)
Comment (raquelarrojo) [reply website/screens/components/flex/specs/FlexSpecsPage.tsx]: Done.

File: website/screens/components/flex/usage/FlexUsagePage.tsx (hunk 2 above)
Comment (raquelarrojo) [reply website/screens/components/flex/usage/FlexUsagePage.tsx]: Done

File: website/screens/components/flex/usage/FlexUsagePage.tsx (hunk 3 above)
Comment (raquelarrojo) [reply website/screens/components/flex/usage/FlexUsagePage.tsx]: Done

File: website/screens/components/flex/usage/FlexUsagePage.tsx (hunk 4 above)
Comment (raquelarrojo) [reply website/screens/components/flex/usage/FlexUsagePage.tsx]: Done

**Review by Lisa159 — APPROVED** [review]

**Pull request merged**